apps/aecore/src/aec_governance.erl

```diff
@@ -16,7 +16,8 @@ blocks_to_check_difficulty_count() ->
     ?BLOCKS_TO_CHECK_DIFFICULTY_COUNT.
 
 expected_block_mine_rate() ->
-    ?EXPECTED_BLOCK_MINE_RATE.
+    application:get_env(aecore, expected_mine_rate,
```

Comment:
I am a bit concerned in using application config for testing-motivated change. Have you considered using a fully black-box approach i.e.:
* Disable the ntpd daemon (or whatever sets continuously the OS time - if there is any such thing); and
* Before starting test, start concurrent OS process that loops full speed setting, at each loop, time to 10 seconds (or whatever needed) forward. This shall cause the timestamp in each new block to be large enough to keep difficulty low. (This of course implies reviewing how the time is retrieved for building block candidate, that shall be - I believe - OS time.)

Reply:
I found it useful to be able to configure it, esp. dev1-3 while manually testing.
As for modifying OS time - how would that play with Travis?

Reply:
> As for modifying OS time - how would that play with Travis?

I would expect modifying OS time meaningful only on disposable host e.g. Travis (it would cause some problems on development laptop...). It would need to be tested. I just wanted to make sure we thought of all options.